tests/test_array.py

```diff
@@ -746,6 +746,43 @@ def test_resize_2d(store: MemoryStore, zarr_format: ZarrFormat) -> None:
     assert new_shape == z[:].shape
 
 
+@pytest.mark.parametrize("store", ["local"], indirect=True)
+@pytest.mark.parametrize("open", ["open", "open_array"])
+def test_append_config_passed(store: LocalStore, open: str, zarr_format: ZarrFormat) -> None:
```

Comment:
Why do our existing `write_empty_chunks` not work here?

Reply:
for context, it would probably be preferable to expand the scope of an existing test instead of adding a new one